src/data/prepopulated.ts

```diff
@@ -255,4 +255,18 @@ export const prepopulated: {
     yourMask: 'basic',
     voice: 'silent',
   },
+  [i18n.t('scenario.custom')]: {
```

Comment:
Is it necessary to create a dummy scenario here? Seems like an implementation detail of SavedDataSelector.tsx, could it be inserted there?

Reply:
I think so, actually. I'm not sure how else to reset the rest of the form. The comment is inaccurate and I'll remove it.

Reply:
Oh, neat, the comment made me think it was no-op, but clearing the form is neat.